libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html

```diff
@@ -21,162 +21,112 @@
-          @if (formGroup.controls.dateRange.errors?.resolutionTransition) {
-            <watt-validation-message
-              size="normal"
-              type="danger"
-              icon="danger"
-              message="{{
-                t('create.quarterlyResolutionTransitionError', {
-                  resolutionTransitionDate: resolutionTransitionDate | wattDate
-                })
-              }}"
-            />
-          } @else if (
-            latestPeriodEnd && formGroup.controls.calculationType.value === "BALANCE_FIXING"
-          ) {
-            <watt-validation-message
-              size="normal"
-              type="warning"
-              icon="warning"
-              message="{{
-                t('create.periodWarning', { latestPeriodEnd: latestPeriodEnd | wattDate })
-              }}"
-            />
-          }
-
-          <!-- Execution type -->
-          <div class="chips">
-            <label>{{ t("create.executionType") }}</label>
-            <watt-filter-chip
-              choice
-              name="executionType"
-              value="ACTUAL"
-              [selected]="true"
-              (selectionChange)="onExecutionTypeSelected($event)"
-            >
-              {{ t("executionTypes.ACTUAL") }}
-            </watt-filter-chip>
-            <watt-filter-chip
-              choice
-              name="executionType"
-              value="SIMULATION"
-              [disabled]="true"
-              (selectionChange)="onExecutionTypeSelected($event)"
-            >
-              {{ t("executionTypes.SIMULATION") }}
-            </watt-filter-chip>
-          </div>
-
-          <!-- scheduling -->
-          <div class="chips">
-            <label>{{ t("create.scheduling") }}</label>
-            <watt-filter-chip choice name="schedule" value="NOW" [selected]="true">
-              {{ t("scheduling.NOW") }}
-            </watt-filter-chip>
-            <watt-filter-chip choice name="schedule" value="SCHEDULED" [disabled]="true">
-              {{ t("scheduling.SCHEDULED") }}
-            </watt-filter-chip>
-          </div>
+    <form
+      id="create-calculation"
+      class="grid"
+      novalidate
+      [formGroup]="formGroup"
+      (ngSubmit)="
+        latestPeriodEnd &&
+        formGroup.controls.calculationType.value === CalculationType.BalanceFixing
+          ? (showPeriodWarning = true)
+          : modal.close(true)
+      "
+    >
+      <!-- Quarterly Resolution Transition Error -->
+      @if (formGroup.controls.dateRange.errors?.resolutionTransition) {
+        <watt-validation-message
+          size="normal"
+          type="danger"
+          icon="danger"
+          message="{{
+            t('create.quarterlyResolutionTransitionError', {
+              resolutionTransitionDate: resolutionTransitionDate | wattDate
+            })
+          }}"
+        />
+      } @else if (
+        latestPeriodEnd && formGroup.controls.calculationType.value === "BALANCE_FIXING"
```

Comment:
Suggestion: Do the same here :-)

Reply:
b0a7711c7f0e94b92570696dac1126837411c45f